pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.towncrier]
+# Read https://github.com/ethereum/lahja/newsfragments/README.md for instructions
+package = "lahja"
+filename = "docs/release_notes.rst"
+directory = "newsfragments"
+underlines = ["-", "~", "^"]
+issue_format = "`#{issue} <https://github.com/ethereum/lahja/issues/{issue}>`__" 
```

Comment:
*nitpick*: Can you make sure to add a new line (as github suggests ![image](https://user-images.githubusercontent.com/521109/68601107-fc761980-04a3-11ea-9da0-a358263b7b38.png))